adapter/vppapiclient/vppapiclient_adapter.go

```diff
@@ -19,14 +19,15 @@
 package vppapiclient
 
 /*
-#cgo CFLAGS: -DPNG_DEBUG=1
+#cgo CFLAGS: -DPNG_DEBUG=1 -I /opt/vpp-agent/dev/vpp/src
```

Comment:
This is temporarily required until <vpp/stats/stat_segment.h> is included in vpp-dev package.